### DISCUSSION TITLE
SVG while importing BPMN file using `bpmn-client`

### DISCUSSION THREAD

**Discussion by MaheshkumarSundaram** [bpmnServer/bpmn-server#106] [General]
Hi @ralphhanna 

Currently, we are able to save the BPMN file using `server.definitions.import` from `bpmn-client`.
But we can't save the corresponding SVG of that BPMN file along with it.

Could I please request you to add an option to save SVG as well along with BPMN?

Thank you!

**Comment by ralphhanna** [comment]
Done, in next release 1.3.26

client.definitions.import now takes two file paths 
```
    async import(name, pathToBPMN,pathToSVG=null) 
```
